chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -424,6 +424,8 @@
     Provisioner::Repositories.inspect_repos(node)
     repos = Provisioner::Repositories.get_repos(node, "suse", target_platform_version)
 
+    packages = node[:provisioner][:packages][os] || []
```

Comment:
Thanks to @rsalevsky I just realized that this does not work anymore when installing crowbar from scratch.
During refactoring I accidently changed this line so that it depends on `[:packages]` to exist which it does not initially (`"required": false` and no default set)

Creating a follow-up ... (and will add link to it here as soon as it's pushed)